### PR TITLE
fix(TDI-47519): Kafka ssl conflict with schema registry

### PR DESCRIPTION
**Pull request by olhava**
* add migration task

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by Tanya5091 — APPROVED** [review]
LGTM

**Review by Tanya5091 — DISMISSED** [review]

**Review by Tanya5091 — APPROVED** [review]

**Review by Tanya5091 — APPROVED** [review]

**Review by Tanya5091 — APPROVED** [review]

File: main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/SetKeystoreKafkaMigrationTask.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        List<String> kafkaSslComponents = Arrays.asList("tKafkaConnection", "tKafkaInput", "tKafkaOutput");
+        List<String> schemaRegistrySslComponents = Arrays.asList("tKafkaInput", "tKafkaOutput");
+
+        boolean modified = false;
+
+        // schema registry with reused kafka ssl and empty tSetKeystore
+        for (String componentName : schemaRegistrySslComponents) {
+            IComponentFilter componentFilter = new NameComponentFilter(componentName);
+            try {
+                modified = ModifyComponentsAction.searchAndModify(item, processType, componentFilter,
```
Comment (ozhelezniak-talend) [inline main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/SetKeystoreKafkaMigrationTask.java:64]: modified |=

Comment (ozhelezniak-talend) [reply main/plugins/org.talend.repository/src/main/java/org/talend/repository/model/migration/SetKeystoreKafkaMigrationTask.java:64]: in tree places

**Review by ozhelezniak-talend — CHANGES_REQUESTED** [review]

**Review by ozhelezniak-talend — DISMISSED** [review]
lgtm

**Review by Tanya5091 — DISMISSED** [review]

**Review by ozhelezniak-talend — APPROVED** [review]
lgtm

**Pull request merged**